src/main/java/com/nvs/store/models/cart/CartItem.java

```diff
@@ -0,0 +1,28 @@
+package com.nvs.store.models.cart;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.nvs.store.models.product.Product;
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Table(name = "cart_items")
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CartItem {
+    @Id
+    @GeneratedValue
+    private Long id;
+    @JsonIgnore
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "cart_id")
+    @ToString.Exclude
+    private Cart cart;
+    @OneToOne
```

Comment:
Looks like the relation between CartItem and Product is many-to-one, assume that one product can be included to many carts. Even user can manage single cart at the same point in time, other users may decide to include the product to their own cart.

Reply:
fixed ✅